ckanext/datajson/harvester_base.py

```diff
@@ -401,6 +366,59 @@ def _size_check(self, key, value):
             raise DataError('%s: Maximum allowed size is %i. Actual size is %i.' % (
                     key, MAX_SIZE, len(value)
             ))
+    
+    def get_harvest_source_id(self, package_id):
+        harvest_object = model.Session.query(HarvestObject) \
+            .filter(HarvestObject.package_id == package_id) \
+            .filter(HarvestObject.current==True).first()
+
+        return harvest_object.source.id if harvest_object else None
+
+    def is_part_of_to_package_id(self, ipo, harvest_object):
```

Comment:
Would you please add unit tests for these methods?

Reply:
[This commit](https://github.com/GSA/ckanext-datajson/pull/66/commits/2fd993323b22800e82595a456171460a4a801c00) add some unit tests. I will add more